src/Illuminate/Console/Scheduling/ScheduleRunCommand.php

```diff
@@ -60,11 +57,14 @@ public function __construct(Schedule $schedule)
     /**
      * Execute the console command.
      *
+     * @param  \Illuminate\Console\Scheduling\Schedule $schedule
      * @return void
      */
-    public function handle()
+    public function handle(Schedule $schedule)
     {
-        foreach ($this->schedule->dueEvents($this->laravel) as $event) {
+        $this->schedule = $schedule;
+
+        foreach ($schedule->dueEvents($this->laravel) as $event) {
```

Comment:
Please reference the set schedule.

```suggestion
        foreach ($this->schedule->dueEvents($this->laravel) as $event) {
```